xml/System.IO/Directory.xml

```diff
@@ -1039,12 +1080,63 @@ Directory::CreateDirectory("Public\\Html");
         <Parameter Name="enumerationOptions" Type="System.IO.EnumerationOptions" Index="2" FrameworkAlternate="netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="path">To be added.</param>
-        <param name="searchPattern">To be added.</param>
-        <param name="enumerationOptions">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="path">The relative or absolute path to the directory to search. This string is not case-sensitive.</param>
+        <param name="searchPattern">The search string to match against the names of files in <paramref name="path" />.  This parameter can contain a combination of valid literal path and wildcard (* and ?) characters, but it doesn't support regular expressions.</param>
+        <param name="enumerationOptions">An object that describes the search and enumeration configuration to use.</param>
```

Comment:
I'm wondering if you wanna describe this a bit more since this object either here on the remarks

Reply:
I recently documented the EnumerationOptions class. I think the user can click on the link to see what it does. Is that ok?